src/util.ts

```diff
@@ -0,0 +1,16 @@
+const countPrefix = 'count:'
+/**
+ * Returns whether the query specifies a count. Search queries break when count is specified twice.
+ */
+export function queryHasCount(query: string): boolean {
```

Comment:
This would break for search queries where count is escaped:

https://sourcegraph.com/search?q=context:global+%22count:10%22&patternType=literal

Reply:
I don't think it would, check the newly added test case. This function checks any whitespaced-separated component starts with `count:`, which it wouldn't if it's quoted. We [can still use](https://sourcegraph.com/search?q=context:global+%22count:10%22+count:2&patternType=literal) the `count` filter type on top of that, so it should return `false`:

Reply:
Right, my point was that just splitting is not enough. "foo count:4 " will not work in this case. The search team must have a more complicated regex parser that we can use.